src/components/Home/Home.js

```diff
@@ -111,6 +111,28 @@ const Home = () => {
         .then((res) => res.json())
         .then((data) => {
           console.log(data);
+          const arena = data?.centers?.map((res) => ({
+            name: res?.name,
```

Comment:
@Justinnn07 why the word arena, can you please change this to something meaningful like pincodeData, Please change this wherever you have used arena!

Reply:
💯🚀